atom/browser/native_window.cc

```diff
@@ -201,6 +201,13 @@ void NativeWindow::InitFromOptions(const mate::Dictionary& options) {
   options.Get(options::kShow, &show);
   if (show)
     Show();
+
+#if defined(OS_MACOSX)
```

Comment:
Super minor, this should be fine to call on all platforms since it is a no-op on Windows/Linux, but some of the mac-specific options are set directly in the `NativeWindowMac::NativeWindowMac` constructor, do you think this should move there?